libs/ui/src/precinct_scanner_ballot_count_report.test.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { electionSampleDefinition } from '@votingworks/fixtures';
+import { ALL_PRECINCTS_SELECTION } from '@votingworks/utils';
+
+import { PrecinctScannerBallotCountReport } from './precinct_scanner_ballot_count_report';
+
+const pollsTransitionedTime = new Date(2021, 8, 19, 11, 5).getTime();
+const currentTime = new Date(2021, 8, 19, 11, 6).getTime();
+
+test('renders info properly', () => {
+  render(
+    <PrecinctScannerBallotCountReport
+      electionDefinition={electionSampleDefinition}
+      precinctSelection={ALL_PRECINCTS_SELECTION}
+      totalBallotsScanned={23}
+      pollsTransition="pause_polls"
+      pollsTransitionedTime={pollsTransitionedTime}
+      currentTime={currentTime}
+      isLiveMode={false}
+      precinctScannerMachineId="SC-01-000"
+    />
+  );
+
+  // Check header
+  screen.getByText('TEST Polls Paused Report for All Precincts');
+  const electionTitle = screen.getByText('General Election:');
+  expect(electionTitle.parentElement).toHaveTextContent(
+    'General Election: Tuesday, November 3, 2020, Franklin County, State of Hamilton'
```

Comment:
It seems weird to me to have the election name styled as a label for the rest of the election details ("<Election name>: <date, count, state>"), while the other things in the report styled that way are actually labels (e.g. "Polls Paused" and "Scanner ID")

Reply:
I'll leave this as is, but noting the feedback for @beausmith.